nnvm/src/top/tensor/reduce.cc

```diff
@@ -233,6 +255,41 @@ NNVM_REGISTER_REDUCE_OP(min)
     };
 });
 
+NNVM_REGISTER_COLLAPSE_OP(sum)
+.describe(R"code(Reduces lhs to the shape of rhs via sum)code" NNVM_ADD_FILELINE)
+.set_attr<FTVMCompute>(
+  "FTVMCompute", [](const NodeAttrs& attrs,
```

Comment:
Can we move computation part into tvm? It's valuable to pack it first in tvm and then directly call in nnvm.

Reply:
Apparently not. Moving the entire op into topi yields `RuntimeError: Not yet support ewise after reduce`